datafusion/physical-plan/src/stream.rs

```diff
@@ -156,14 +156,62 @@ impl<O: Send + 'static> ReceiverStreamBuilder<O> {
     }
 }
 
-/// Builder for [`RecordBatchReceiverStream`] that propagates errors
+/// Builder for `RecordBatchReceiverStream` that propagates errors
```

Comment:
This `[]` is removed because there is no item named `RecordBatchReceiverStream`, the `RecordBatchReceiverStreamBuilder::build()` will return a trait object rather than a concrete type.

Reply:
Oops, I just realized that this item exists, it is just hidden from the documents:

```rs
#[doc(hidden)]
pub struct RecordBatchReceiverStream {}
```